README.md

```diff
@@ -0,0 +1,9 @@
+# Wordpress Bedrock Integrations
+
+[![Unsupported](https://img.shields.io/badge/Pantheon-Unsupported-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#unsupported)
+
+## About
+This repo contains settings, configuration, and functionality to enable a WordPress site to work on Pantheon. 
+
+## Installation
+Add this to a project via composer: `composer require pantheon-systems/wordpress-bedrock-integrations`.
```

Comment:
Not certain about including this line, as installing this repo as is doesn't actually do anything.